opendevin/runtime/plugins/agent_skills/setup.sh

```diff
@@ -4,10 +4,10 @@ set -e
 
 # add agent_skills to PATH
 echo 'export PATH=/opendevin/plugins/agent_skills:$PATH' >> ~/.bashrc
-export PATH=/opendevin/plugins/agent_skills:$PATH
 
 # add agent_skills to PYTHONPATH
 echo 'export PYTHONPATH=/opendevin/plugins/agent_skills:$PYTHONPATH' >> ~/.bashrc
-export PYTHONPATH=/opendevin/plugins/agent_skills:$PYTHONPATH
 
-pip install flake8 python-docx PyPDF2 python-pptx pylatexenc openai opencv-python
+source ~/.bashrc
+
+/opendevin/miniforge3/bin/pip install flake8 python-docx PyPDF2 python-pptx pylatexenc openai opencv-python
```

Comment:
Perhaps using `pip` would be better? Older sandbox image do not have /opendevin/miniforge3/bin/pip.

Reply:
But going forward, we are going to use the new sandbox so i think it shouldn't be a big problem

Reply:
Another reason here is that i'm worried about is that, hopefully the agent is not messing around the same python environment that `od-runtime-client` is using..

Reply:
This line whill cause integration test failed. 
```
RuntimeError: Failed to initialize plugin agent_skills with exit code 127 and output /opendevin/plugins/agent_skills/setup.sh: line 13: /opendevin/miniforge3/bin/pip: No such file or directory

```

Reply:
NVM, find it is image version problem.